docs/updating.rst

```diff
@@ -2,35 +2,55 @@
 How to update the Atomic Developer Bundle to the latest version
 ===============================================================
 
-* If you are using the image from atlas.hashicorp.com
+Depending on your implementation of the ADB, you might be able to directly
+update your Vagrant box. In some cases, you will need to perform a clean
+installation or follow additional steps to complete the update.
```

Comment:
It looks like Naina confirmed that this step is required for CDK, which means only CentOS images. Will adjust accordingly.